### PR TITLE
Aftershock and Dinomod Mutagen fixes

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Fixes mutagen in Aftershock and Dinomod

#### Describe the solution
Mirrors #61956

#### Describe alternatives you've considered
none

### CODE REVIEW

File: data/mods/Aftershock/items/mutagen.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
     "name": "mi-go mutagen",
     "description": "An extremely rare mutagen cocktail, it smells of sulphur and glows orange.",
     "price": 500000,
-    "vitamins": [ [ "mutagen_migo", 175 ], [ "mutagen", 125 ] ]
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You drink the mana mutagen.",
```
Comment (Knut-Aage-Hofseth) [inline data/mods/Aftershock/items/mutagen.json:27]:
```suggestion
      "activation_message": "You drink the mi-go mutagen.",
```

File: data/mods/Aftershock/items/mutagen.json [hunk 2]
```diff
@@ -47,6 +51,10 @@
     "name": { "str": "mastodon mutagen" },
     "description": "An extremely rare mutagen cocktail, it seeks to cure what's deep inside.",
     "price": 500000,
-    "vitamins": [ [ "mutagen_mastodon", 175 ], [ "mutagen", 125 ] ]
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You drink the mana mutagen.",
```
Comment (Knut-Aage-Hofseth) [inline data/mods/Aftershock/items/mutagen.json:56]:
```suggestion
      "activation_message": "You drink the mastodon mutagen.",
```

File: data/mods/DinoMod/items/mutagen.json [hunk 3]
```diff
@@ -23,7 +23,11 @@
     "description": "An extremely rare mutagen cocktail, it smells like blood and is a green goop with blood-red streaks",
     "color": "light_green",
     "price": 500000,
-    "vitamins": [ [ "mutagen_tyrant", 175 ], [ "mutagen", 125 ] ]
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You drink the mana mutagen.",
```
Comment (Knut-Aage-Hofseth) [inline data/mods/DinoMod/items/mutagen.json:28]:
```suggestion
      "activation_message": "You drink the tyrant mutagen.",
```

File: data/mods/DinoMod/items/mutagen.json [hunk 4]
```diff
@@ -49,7 +53,11 @@
     "description": "An extremely rare mutagen cocktail, it seeks to cure what's deep inside.",
     "color": "light_green",
     "price": 500000,
-    "vitamins": [ [ "mutagen_stego", 175 ], [ "mutagen", 125 ] ]
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You drink the mana mutagen.",
```
Comment (Knut-Aage-Hofseth) [inline data/mods/DinoMod/items/mutagen.json:58]:
```suggestion
      "activation_message": "You drink the stego mutagen.",
```

File: data/mods/DinoMod/items/mutagen.json [hunk 5]
```diff
@@ -75,6 +83,10 @@
     "description": "An extremely rare mutagen cocktail containing a mix of biological samples, spiked with rum.",
     "color": "light_green",
     "price": 500000,
-    "vitamins": [ [ "mutagen_horns", 175 ], [ "mutagen", 125 ] ]
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You drink the mana mutagen.",
```
Comment (Knut-Aage-Hofseth) [inline data/mods/DinoMod/items/mutagen.json:88]:
```suggestion
      "activation_message": "You drink the horns mutagen.",
```

**Review by Knut-Aage-Hofseth — COMMENTED** [review]
The activation message was kept and talked about drinking mana mutagen. Change suggestion is for updating the activation message to mention the correct mutagen.

**Comment by LyleSY** [comment]
Looks good once those changes are made

**Pull request merged**